composer.json

```diff
@@ -14,13 +14,12 @@
         "issues": "https://github.com/fastly/fastly-php/issues",
         "source": "https://github.com/fastly/fastly-php"
     },
-    "minimum-stability": "dev",
     "require": {
         "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "~6.0@dev"
+        "guzzlehttp/guzzle": "^6.2.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "~2.0"
+        "phpspec/phpspec": "^2.5.0"
```

Comment:
@thommahoney 

This should fix the issues observed in https://github.com/fastly/fastly-php/pull/21.